src/utils/constants.ts

```diff
@@ -58,12 +59,13 @@ export enum ChainId {
     GNOSIS_CHAIN = 100,
     POLYGON = 137,
     ZKSYNC_TESTNET = 280,
+    FANTOM = 250,
     ZKSYNC = 324,
     ZKEVM = 1101,
     ARBITRUM_ONE = 42161,
     AVALANCHE = 43114,
     BASE_GOERLI = 84531,
-    FANTOM = 250,
+    SEPOLIA = 11155111,
 }
```

Comment:
Maybe this could be sorted in alphabetical order instead of numerical?

Reply:
I don't have a strong opinion on this, but I'd say it's ok as is.